src/move/move.ts

```diff
@@ -0,0 +1,63 @@
+import { Board } from "../board/board";
+import { GameState } from "../game/state";
+import { NEUTRALE_TILE, findTile, findTileByCoordinate } from "../tile/tile";
+
+interface Action {
+  value: any;
+}
+
+interface UserMove {
+  gameState: GameState;
+  allowedMove: boolean;
+}
```

Comment:
```suggestion
interface CheckedUserMove {
  gameState: GameState;
  allowedMove: boolean;
}
```

(but this is just a suggestion)

Reply:
Done